sale_order_line_price_history/migrations/13.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,16 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE sale_order_line sol
+        SET order_partner_id = so.partner_id
+        FROM sale_order so
+        WHERE so.id = sol.order_id AND sol.order_partner_id <> so.partner_id
```

Comment:
```suggestion
        WHERE so.id = sol.order_id AND sol.order_partner_id != so.partner_id
```

Reply:
Transact-SQL ISO standard not equal operator is `<>`
Why do you want us to use an alias?
Is it because it looks more like python?

Reply:
Yes, for homogeneity, but honestly I didn't know about the standard being `<>` in SQL...